### PR TITLE
MISP-1793: chipageddon support old and new wifi modules (S5R1)

### PR DESCRIPTION
**Pull request by shanx**

### CODE REVIEW

**Review by bryanfoley — CHANGES_REQUESTED** [review]
The commit that gitmodule points to needs to be updated to the same as the R2 gitmodule (the new realtek driver files are behind that commit) and the kernel config file needs to be regenerated using the 'menuconfig' option of the build.sh script to enable full support for the new wifi module. I'll make the changes.

**Comment by shanx** [comment]
I went: 'te kort door de bocht' :) Thanks!

File: configs/opinicus_config [hunk 1]
```diff
@@ -1849,6 +1849,7 @@ CONFIG_RTLWIFI_USB=m
 CONFIG_RTL8192C_COMMON=m
 CONFIG_RTL8XXXU=m
 CONFIG_RTL8XXXU_UNTESTED=y
+CONFIG_RTL8822CE=m
```
Comment (robinmdh) [inline configs/opinicus_config:1852]: I think you also need the  CONFIG_WLAN_VENDOR_REALTEK=y key and this was not done with menuconfig which I do recommend so you don't miss things like this...

**Comment by robinmdh** [comment]
It turns out the driver only works via PCI and there is no PCI in the R1 (wifi module is connected via USB)
So this can't be supported.

File: configs/opinicus_config (hunk 1 above)
Comment (shanx) [reply configs/opinicus_config:1852]: That was already in the file

**Pull request closed without merging**